BazelExtensions/workspace.bzl

```diff
@@ -150,8 +150,14 @@ def _impl(repository_ctx):
 
     # For now, we curl the podspec url before the script runs
```

Comment:
Is it possible to make this work the same way as the python version? We introspective the URL to see if it’s a relative file path and use that if it’s provided:
https://github.com/pinterest/PodToBUILD/blob/49065b30d1ada4f09caefa773555f132ba4c65b0/bin/update_pods.py#L249